test/test_quiz_mode.py

```diff
@@ -67,6 +67,69 @@ def test_passed_quiz_normal_task(self):
         task = json.loads(response.data)
         assert any(task['id'] == non_golden_task.id for non_golden_task in non_golden_tasks)
 
+    @with_context
+    def test_project_quiz_enabled_and_user_quiz_enabled_result_quiz_mode_enabled(self):
+        '''Test that the user can see quiz mode if the user is In Progress for quiz mode and quiz mode is enabled for the project'''
+        '''See also: pybossa/api/__init__.py line 270
+           quiz_mode_enabled = user.get_quiz_in_progress(project) and project.info["quiz"]["enabled"]'''
+        project, user = self.create_project_and_user()
+
+        # Enable quiz mode on the project.
+        quiz = project.get_quiz()
+        quiz["enabled"] = True
+        project.set_quiz(quiz)
+
+        project.info["quiz"]["enabled"] = True
```

Comment:
I don't think this is needed line 80 should be setting it already

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/419/commits/243b8a5527585375db601ba2dd69a47c1dd8b601
Thank you. :)